packages/test-runner/src/cli.ts

```diff
@@ -100,10 +100,18 @@ async function runStage1(command) {
       process.exit(1);
     }
   });
-
+  const reporter = new Multiplexer(reporterObjects);
+  runner = new Runner(config, reporter);
+  let files;
+  try {
+    files = collectFiles(testDir, '', filteredArguments.slice(1), command.testMatch, command.testIgnore);
+  } catch (error) {
+    reporter.onLoadingFailure(testDir, error.message);
```

Comment:
Nit: should we move collection into the runner? It is a bit weird that CLI talks to the reporter, I would expect only runner to do that.

Reply:
yes